Assets/HoloToolkit/Utilities/Prefabs/SceneLauncher.prefab

```diff
@@ -225,21 +225,17 @@ TextMesh:
   m_PrefabParentObject: {fileID: 0}
   m_PrefabInternal: {fileID: 100100000}
   m_GameObject: {fileID: 1177239735944006}
-  m_Text: 'Air tap a scene to load it.
-
-    When finished with a scene, say "Scene Launcher" or press the Backspace key to
-    return the this scene list.
-
-    Edit the scenes in this list in Unity > File > Build Settings > Scenes in Build.
-
-    Make sure that this scene (SceneLauncher) is the first scene (#0).
-
-    If you would like to use all scenes in the project, search for t:Scene in the
-    Project tab, select all, and drag all scenes into Scenes in Build.'
+  m_Text: "Air tap a scene to load it.\nWhen finished with a scene, say \"Scene Launcher\"\nor
```

Comment:
For me, this text overlaps with the scene launcher. Too many \n newlines? It seems to line up better if there is just one carriage return after each sentence. Personally, I don't mind having to scroll way over to read lengthy text--I might even prefer that to making the block much taller.

Here is what I'm currently seeing:
![instructionsoverlap](https://user-images.githubusercontent.com/17389739/30195101-41dc0648-940c-11e7-94db-3a8f8aaafbba.png)

Reply:
😂😂😂 I can't win

Reply:
I didn't think about what it looks like in the scene after the buttons got generated. I'll have a fix in a bit after dinner.